addons/website_event/views/event_event_add.xml

```diff
@@ -4,17 +4,20 @@
 <record id="event_event_view_form_add" model="ir.ui.view">
     <field name="name">event.event.view.form.add</field>
     <field name="model">event.event</field>
+    <field name="mode">primary</field>
+    <field name="inherit_id" ref="event.view_event_form"/>
     <field name="arch" type="xml">
-        <form js_class="website_new_content_form">
-            <group>
-                <field name="website_url" invisible="1"/>
-                <field name="company_id" invisible="1"/>
-                <field name="name" placeholder="e.g. &quot;Conference for Architects&quot;" string="Event Name"/>
-                <field name="address_id" context="{'show_address': 1}"/>
-                <field name="date_begin" string="Start &#8594; End" widget="daterange" options="{'end_date_field': 'date_end'}" />
-                <field name="date_end" invisible="1" />
-            </group>
-        </form>
+        <xpath expr="//form" position="attributes">
+            <attribute name="js_class">website_new_content_form</attribute>
+        </xpath>
+        <xpath expr="//header" position="replace"/>
+        <xpath expr="//div[hasclass('oe_button_box')]" position="replace"/>
+        <xpath expr="//field[@name='kanban_state']" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </xpath>
+        <xpath expr="//form" position="inside">
+            <field name="website_url" invisible="1"/>
+        </xpath>
```

Comment:
Just a question why are we adding website_url over here 
I know it was done previously :thinking:

Reply:
`website_url` is used to redirect user to that event page in edit mode
REF - We need [compute path](https://github.com/odoo/odoo/blob/e1b3bdb72c25b33bc4a4fb951c593043eaa4ea28/addons/website/static/src/systray_items/new_content.js#L270-L271) in New Content
and compute path function needs `website_url`  -> [Here](https://github.com/odoo/odoo/blob/e1b3bdb72c25b33bc4a4fb951c593043eaa4ea28/addons/website/static/src/js/new_content_form.js#L21-L22)

Reply:
Hello @yagp-odoo 
Can you check this 
https://youtu.be/Q_DzWX8TpQY

Reply:
did not tried with other url, but when I removed it, I was redirected to Home`(/)`